docs/marionette.region.md

```diff
@@ -257,18 +261,22 @@ This can be useful in unit testing your views.
 Override the region's `attachHtml` method to change how the view is attached
 to the DOM. This method receives one parameter - the view to show.
 
-The default implementation of `attachHtml` is:
+The default implementation of `attachHtml` works a bit like the following:
 
 ```javascript
 var Mn = require('backbone.marionette');
```

Comment:
I think this particular example is easy enough to keep in the docs.

I do agree that we should get some examples in place, however.